R/input-time-series.R

```diff
@@ -58,12 +58,15 @@ get_art_metadata <- function(art) {
 
   # plot types below are derived from art_current which is always assumed
   # to be present
-  plot_types <- c(get_strats_for_plot_type("art"), "art_adult_sex_ratio", "art_child_adult_ratio")
+  plot_types <- c("art_current", get_strats_for_plot_type("art", c("adult","adult_f", "adult_m", "child")),
+                  "art_current_adjusted", get_strats_for_plot_type("art_adjusted", c("adult","adult_f", "adult_m", "child")),
```

Comment:
another optional change, feels more consistent to have the `current`, `adult`, `adult_f`, etc at the end but up to you
```suggestion
                  "art_adjusted_current", get_strats_for_plot_type("art_adjusted", c("adult","adult_f", "adult_m", "child")),
```